### ISSUE TITLE
Reporting False Positive

### ISSUE THREAD

**Issue by technicgeek** [nextdns/metadata#747]
Hello,

The following domain does not play video:

https://weather.com/news/weather/video/time-lapse-video-shows-california-wildfire-consuming-camera?par=samsung_widget_ILO

Thanks

**Comment by lyrad87** [comment]
Plays just fine for me. Maybe one of your blocklists is the culprit, so please share which ones you use. Then we can try to help find which domain is getting blocked and you can report it upstream. NextDNS most likely won't fix it on their side, so it's either the upstream report or you have to add the domain to your Allowlist.

**Comment by technicgeek** [comment]
@lyrad87 This is strange. I only used NextDNS default blocklist. I disabled it and all worked this time. I then re-enabled it and there was no difference. Domain works still. Maybe someone saw this post and allowed it?

**Comment by crssi** [comment]
Very likely due to blocked `securepubads.g.doubleclick.net`... but that doesn't mean I would unblock this shit.
